specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-08-03-preview/location.json

```diff
@@ -0,0 +1,184 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ContainerServiceClient",
+    "description": "The Container Service Client.",
+    "version": "2022-08-03-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerService/locations/{location}/orchestrators": {
```

Comment:
Should we use this chance to update the API to list supported k8s versions in a region? I think it was named list orchestrators before when ACS supported other orchestrators in addition to Kubernetes, but it isn't the case anymore.

Reply:
Yes, that's the problem, now that there are concerns, I'll cancel this PR until a consensus is reached.